graylog2-web-interface/src/components/welcome/Welcome.tsx

```diff
@@ -31,8 +33,26 @@ const StyledSectionComponent = styled(SectionComponent)`
   flex-grow: 1;
 `;
 
+const ChangeStartPageHelper = ({ readOnly, userId }: { readOnly: boolean, userId: string}) => {
+  if (readOnly) {
+    return (
+      <span>
+        {' '}
+        If you like, you can ask administrator to change your personal start page
```

Comment:
As far as I know it is not possible to define a start page for a read only user.

Reply:
In every case we should add a dot at the end of the sentence.